modules/openapi-generator/src/main/resources/php-slim4-server/composer.mustache

```diff
@@ -9,7 +9,7 @@
   "require": {
     "php": "^7.1",
     "slim/slim": "^4.0",
-    "dyorg/slim-token-authentication": "dev-slim4",
```

Comment:
This branch/release does not exist per our CI error message and:
https://packagist.org/packages/dyorg/slim-token-authentication

Reply:
Per here: https://vvv.tobiassjosten.net/php/have-composer-use-development-branches/
A dev prefix lets us use a github branch